### PR TITLE
A persona and pathway chapter for the turing way community handbook

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Partially related to #1121 and #1252 

* This PR creates a new chapter called Persona Creation in the project design guide

### List of changes proposed in this PR (pull-request)

* [ ] A landing page for the chapter
* [ ] A subchapter on persona and pathway for new and sustained contributors
* [ ] A subchapter on contributions types and pathways to make those contributors
* [ ] An image with CC-BY license from Scriberia+The Turing Way
* [ ] TOC file updated

### What should a reviewer concentrate their feedback on?

- [ ] The steps and content make sense
- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/community-handbook/persona.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+height: 400px
+name: personas
+alt: 4 people with diverse appearances are talking to each other, sharing ideas directly with each other and also sharing on social media.
+---
+People sharing their ideas through direct conversations and on social media. _The Turing Way_ project illustration by Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3695300
+```
+Persona is a detail of an imaginary user or member, based on real-world observations and understandings of existing members or potential future members.
+It is meaningful when paired with pathways through which these personas engage with a project.
+
+To understand how our contributors or users might interact with _The Turing way_, we have developed some personas and Pathways that we are sharing in this chapter.
```
Comment (rainsworth) [inline book/website/community-handbook/persona.md:23]:
```suggestion
To understand how our contributors or users might interact with _The Turing Way_, we have developed some personas and pathways that we are sharing in this chapter.
```

File: book/website/community-handbook/persona.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+They aims to capture different personas of the community members to explore how they come in touch with this project, how they continue to engage and how can we take their experiences and expectations into account when planning our community aspects.
```
Comment (rainsworth) [inline book/website/community-handbook/persona.md:24]:
```suggestion
They aim to capture different personas of the community members to explore how they come into contact with this project, how they continue to engage and how we can take their experiences and expectations into account when planning our community aspects.
```

File: book/website/community-handbook/persona.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+```{figure} ../figures/personas.png
+---
+height: 400px
+name: personas
+alt: 4 people with diverse appearances are talking to each other, sharing ideas directly with each other and also sharing on social media.
+---
+People sharing their ideas through direct conversations and on social media. _The Turing Way_ project illustration by Scriberia. Zenodo. http://doi.org/10.5281/zenodo.3695300
+```
+Persona is a detail of an imaginary user or member, based on real-world observations and understandings of existing members or potential future members.
```
Comment (rainsworth) [inline book/website/community-handbook/persona.md:20]:
```suggestion
A persona is a detail of an imaginary user or member, based on real-world observations and understandings of existing members or potential future members.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+1. Discovery - How they first hear about the project or group.
+2. First Contact - How they first engage with the project or group, their initial interaction.
+3. Participation - How they first participate or contribute.
+4. Sustained Participation - How their contribution or involvement can continue.
+5. Networked Participation - How they may network within the community.
+6. Leadership - How they may take on some additional responsibility on the project, or begin to lead.
+
+We have created personas based on GitHub skills and willingness to contribute to the project.
+We have also listed a few examples of how our personas might experience their first interactions with our project and the "pathway" they take to get involved.
+For each persona, we highlight how their involvement with the Turing Way can improve and enrich the project and community.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:17]:
```suggestion
For each persona, we highlight how their involvement with _The Turing Way_ can improve and enrich the project and community.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+This document is meant to suggest ways you might interact with, learn from, and contribute to the Turing Way.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:19]:
```suggestion
This document is meant to suggest ways you might interact with, learn from, and contribute to _The Turing Way_.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+4. Noor, someone who doesn't know they want to contribute, but does
+5. Robin, someone who can only contribute outside of their normal working hours
+
+### 1. Sam, a PhD student with no GitHub experience
+
+Sam is a PhD student in biology at University College London.
+They are learning how to code in Python, but have not yet had any training in version control or GitHub.
+They're interested in learning how to use open notebooks such as Jupyter to do their analysis so that they can easily share their work for critique with their very busy supervisor.
+
+1. **Discovery** - Sam hears about the Turing Way Book Dash events through an email from the department's Open Science Champion.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:36]:
```suggestion
1. **Discovery** - Sam hears about _The Turing Way_ Book Dash events through an email from the department's Open Science Champion.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+2. **First Contact** - Sam checks out the GitHub repository to learn more about the project and see what skills are needed that they may be able to contribute.
+3. **Participation** - They attend the London Book Dash, accepted for their enthusiasm to learn how to make their PhD research reproducible. They participate in the discussions, read existing chapters, and scroll through issues to see where they can contribute.
+4. **Sustained Participation** - After gaining experience with GitHub through submitting pull requests and issues during the Book Dash, Sam is empowered to continue contributing to the online discussion and suggesting topics for the book, as well as editing and reviewing existing chapters by applying them to his research.
+5. **Networked Participation** - They encourage some of their fellow PhD students to read the Turing Way and start using GitHub.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:40]:
```suggestion
5. **Networked Participation** - They encourage some of their fellow PhD students to read _The Turing Way_ and start using GitHub.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+6. **Leadership** - Sam submits one of their research projects as a case study for the Turing Way.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:41]:
```suggestion
6. **Leadership** - Sam submits one of their research projects as a case study for _The Turing Way_.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+
+###  2. Alia, a researcher with Git/GitHub experience and book topic expertise
+
+Alia is a postdoc in Morocco who programs in R.
+They are passionate about open data and using data for good, but also understand it can be more complex than that.
+They attend data science meetups in the city but are interested in finding a community working towards improving how research is conducted.
+They have experience with GitHub and contributing to open projects.
+
+1. **Discovery** - Alia first hears about the Turing Way via Twitter ([#TuringWay](https://twitter.com/search?q=%23TuringWay&src=typed_query)).
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:50]:
```suggestion
1. **Discovery** - Alia first hears about _The Turing Way_ via Twitter ([#TuringWay](https://twitter.com/search?q=%23TuringWay&src=typed_query)).
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+2. **First Contact** - They land on the project's README and look for the contributing guidelines to see where their skills can be applied.
+3. **Participation** - They find an issue asking for help on writing the Credit for Reproducible Research chapter and add a few paragraphs on how to properly cite research software.
+4. **Sustained Participation** - After constructive feedback and collaboration with the Turing Way team, Alia follows the community on social media and attend the online Collaboration Cafe.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:53]:
```suggestion
4. **Sustained Participation** - After constructive feedback and collaboration with _The Turing Way_ team, Alia follows the community on social media and attend the online Collaboration Cafe.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+6. **Leadership** - Alia volunteers to write the chapter on "Scoping a data project for RSEs" that they found requested in the [Book Skeleton](https://github.com/alan-turing-institute/the-turing-way/blob/master/book_skeleton.md).
+
+### 3. Amal, a PhD supervisor who is an expert in an aspect of reproducible research
+
+Amal is a PhD supervisor who is an expert in running risk assessments for projects using reproducible research.
+They are always looking for opportunities to share their expertise, particularly with students, as they think the consistent application of best practice is important.
+They are very keen to collaborate with people and to volunteer their time for collaboration projects.
+They like seeing their work make an impact and are keen to know about the eventual result of their collaborations.
+
+1. **Discovery** - Amal finds out about the Turing Way book from the Twitter feed of experts in reproducible research who they follow.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:64]:
```suggestion
1. **Discovery** - Amal finds out about _The Turing Way_ book from the Twitter feed of experts in reproducible research who they follow.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+2. **First Contact** - Amal navigates to the GitHub repository and reads the content there over a couple of days. While reading, they make notes on areas they could add to from their research.
+3. **Participation** - They compile their work into a chapter format and submit it to the repo.
+4. **Sustained Participation** - Amal checks back frequently to look at feedback on their chapter and respond to it. In their spare time, they make suggestions and edits on other chapters.
+5. **Networked Participation** - Amal directs students to their Turing Way chapter when they ask about risk assessments in reproducible research, and begins to direct students to other chapters as part of their teaching.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:68]:
```suggestion
5. **Networked Participation** - Amal directs students to their chapter in the _The Turing Way_ when they ask about risk assessments in reproducible research, and begins to direct students to other chapters as part of their teaching.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 13]
```diff
@@ -0,0 +1,105 @@
...
+6. **Leadership** - Amal promotes the Turing Way at their institution, suggesting to other academics that they get involved with its creation and adopt it as best practice.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:69]:
```suggestion
6. **Leadership** - Amal promotes _The Turing Way_  at their institution, suggesting to other academics that they get involved with its creation and adopt it as best practice.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 14]
```diff
@@ -0,0 +1,105 @@
...
+
+###  4. Noor, a PhD student who is trying to finish their dissertation
+
+Noor is a final-year PhD student who has their dissertation deadline coming up.
+They are feeling the pressure of needing to write up the results of their research but finding it easy to procrastinate.
+They are confident that their research has taught them some particular considerations for reproducible environments, but they are slightly intimidated by the expertise of other people in their field, and they are considering their future after their PhD.
+
+1. **Discovery** - Noor comes across the Turing Way when they're surfing Twitter while trying to write up their research.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:77]:
```suggestion
1. **Discovery** - Noor comes across _The Turing Way_ when they're surfing Twitter while trying to write up their research.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 15]
```diff
@@ -0,0 +1,105 @@
...
+2. **First Contact** - They read the chapter that's relevant to their research and then continue working on their dissertation.
+3. **Participation** - Later, Noor returns to the Turing Way and makes some suggestions and edits to that chapter.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:79]:
```suggestion
3. **Participation** - Later, Noor returns to _The Turing Way_ and makes some suggestions and edits to that chapter.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 16]
```diff
@@ -0,0 +1,105 @@
...
+4. **Sustained Participation** - They return to the repo periodically to read feedback on their commits and make additional comments, restricted to the one chapter they feel they know about.
+5. **Networked Participation** - Noor tweets a link to the Turing Way during a conversation about reproducible research and retweets a response from another expert which leads to a longer conversation.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:81]:
```suggestion
5. **Networked Participation** - Noor tweets a link to _The Turing Way_ during a conversation about reproducible research and retweets a response from another expert which leads to a longer conversation.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 17]
```diff
@@ -0,0 +1,105 @@
...
+2. **First Contact** - Robin was so inspired by the talk that they followed the [Turing Way on Twitter](https://twitter.com/turingway) and starred the [GitHub repository](https://github.com/alan-turing-institute/the-turing-way), to come back and learn more when they had time.
+3. **Participation** - When they find time at the weekend, Robin reads the chapter on Continuous Integration to pick up some tips and tricks - however they see that there isn't yet information on using GitHub Actions. Robyn would like to know more about this topic, so reads tutorials on how to incorporate them into their workflow and opens a [WIP] pull request to add a subsection to the chapter based on what they learn.
+4. **Sustained Participation** - As their contributions are made outside of working hours, the process of learning, writing the subsection and iterating on feedback from the Turing Way team moves at a [relatively] slower pace than someone who can contribute during working hours. Robin appreciates the assistance and communication from the team through the pull request conversation.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:93]:
```suggestion
4. **Sustained Participation** - As their contributions are made outside of working hours, the process of learning, writing the subsection and iterating on feedback from _The Turing Way_ team moves at a [relatively] slower pace than someone who can contribute during working hours. Robin appreciates the assistance and communication from the team through the pull request conversation.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 18]
```diff
@@ -0,0 +1,105 @@
...
+5. **Networked Participation** - Robin mentions the Turing Way and what she learned about using GitHub Actions during one of her team's stand-ups, promoting their use to her colleagues.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:94]:
```suggestion
5. **Networked Participation** - Robin mentions _The Turing Way_ and what she learned about using GitHub Actions during one of her team's stand-ups, promoting their use to her colleagues.
```

File: book/website/community-handbook/persona/persona-contributors.md [hunk 19]
```diff
@@ -0,0 +1,105 @@
...
+6. **Leadership** - Robin gives a lightning talk at a London tech event about GitHub Actions and her contributions to The Turing Way. She praises the inclusiveness of the project and invites more industry experts to check out the resource and open issues on topics that they can contribute to or would like to learn more about.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributors.md:95]:
```suggestion
6. **Leadership** - Robin gives a lightning talk at a London tech event about GitHub Actions and her contributions to _The Turing Way_. She praises the inclusiveness of the project and invites more industry experts to check out the resource and open issues on topics that they can contribute to or would like to learn more about.
```

File: book/website/community-handbook/persona/persona-contributions.md [hunk 20]
```diff
@@ -0,0 +1,309 @@
...
+
+### Representing the Turing Way
+
+Anyone who shares _The Turing Way_ resources in any relevant publication, learning material, conference presentations or community event are acknowledged for representing _The Turing Way_.
+These members may or may not have previously contributed to the project.
+They either volunteer or are recommended by _The Turing Way_ team members for representing this community within or outside the project.
+
+#### Personas & Pathways:
+
+**1.** *Highlighting project resources*: Yehuda used one of the Scriberia and The Turing Way illustrations in a recent talk they gave at an open source workshop that was hosted by their company.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributions.md:221]:
```suggestion
**1.** *Highlighting project resources*: Yehuda used one of the Scriberia and _The Turing Way_ illustrations in a recent talk they gave at an open source workshop that was hosted by their company.
```

File: book/website/community-handbook/persona/persona-contributions.md [hunk 21]
```diff
@@ -0,0 +1,309 @@
...
+They included links to The Turing Way book, GitHub repository and twitter handle.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributions.md:222]:
```suggestion
They included links to _The Turing Way_ book, GitHub repository and Twitter handle.
```

File: book/website/community-handbook/persona/persona-contributions.md [hunk 22]
```diff
@@ -0,0 +1,309 @@
...
+Their slides are available under a CC-BY license and are requested to be linked in a monthly newsletter.
+
+**Acknowledgement:** They are acknowledged in the Contributors table with the 🔊 (`Talk`) emoji.
+
+**2.** *Giving a talk on _The Turing Way_*: Noah is a long time contributor to The Turing Way, they answer questions and review pull requests regularly.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributions.md:227]:
```suggestion
**2.** *Giving a talk on _The Turing Way_*: Noah is a long time contributor to _The Turing Way_, they answer questions and review pull requests regularly.
```

File: book/website/community-handbook/persona/persona-contributions.md [hunk 23]
```diff
@@ -0,0 +1,309 @@
...
+In March 2020, Kirstie - lead developer of The Turing Way - recommended Noah as a suggested speaker to give a talk about the Project Design section of The Turing Way at a conference on open source community management.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributions.md:228]:
```suggestion
In March 2020, Kirstie - lead developer of _The Turing Way_ - recommended Noah as a suggested speaker to give a talk about the Project Design section of _The Turing Way_ at a conference on open source community management.
```

File: book/website/community-handbook/persona/persona-contributions.md [hunk 24]
```diff
@@ -0,0 +1,309 @@
...
+She worked with Noah to practice his presentation in advance of the talk.
+Noah's slides are available under a CC-BY license and are linked from a page in the community handbook.
+
+**Acknowledgement:** They are acknowledged in the Contributors table with the 🔊 (`Talk`), 🚧 , 👀 (`review`), 🤔 (`idea`) and 💬 (`question`) emoji and they will be named as an author on the next release of the book. There is a section of the `CONTRIBUTORS` file saying:
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributions.md:232]:
```suggestion
**Acknowledgement:** They are acknowledged in the Contributors table with the 🔊 (`Talk`), 🚧 , 👀 (`review`), 🤔 (`idea`) and 💬 (`question`) emoji and they will be named as an author on the next release of the book. 
There is a section of the `CONTRIBUTORS` file saying:
```

(New line - also I think something is missing here?)

File: book/website/community-handbook/persona/persona-contributions.md [hunk 25]
```diff
@@ -0,0 +1,309 @@
...
+#### Personas & Pathways:
+
+1. Khasan wrote a relevant online blogpost which was highlighted in the monthly newsletter.
+2. Avaline wrote a research article and cited a chapter from _The Turing Way_, which was then added to the _The Turing Way_ bibliography.
+
+**Acknowledgement:** These contributors are acknowledged with 📝 (`blog`) emoji in the Contributors Table.
+
+### Videos and recordings
+
+**Overview:** We encourage creating video content or animations, record online interactive discussions or link any relevant videos our contributors may have created in the past that can be used as a learning tool and enhance the quality of _The Turing Way_ content.
```
Comment (rainsworth) [inline book/website/community-handbook/persona/persona-contributions.md:276]: Inclusion of `Overview: is a different style to the other sections.

File: book/website/community-handbook/persona/persona-contributors.md (hunk 6 above)
Comment (malvikasharan) [reply book/website/community-handbook/persona/persona-contributors.md:36]: OMG, how could I get this wrong soo many times!? 🙈

File: book/website/community-handbook/persona/persona-contributions.md [hunk 26]
```diff
@@ -0,0 +1,309 @@
...
+
+The various contributions to a chapter are made towards designing, writing, and reviewing its content through GitHub issues, pull requests and reviewing processes.
+
+#### Personas & Pathways:
+
+**1.** *Writing a chapter*: Jordon has written a chapter on "Setting expectations across academia and industry".
+He started writing the chapter at a Turing Way Book Dash event, based on an idea they had developed in advance of the event.
+He responded to review comments and iteratively improved the chapter until the pull request was merged.
+
+**Acknowledgement:** He is acknowledged in the Contributors table with a 👀 (`review`), 🤔 (`idea`) and 🖋 (`content`) emoji.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/community-handbook/persona/persona-contributions.md:96]: Yes, you're right. I might have been very inconsistent with this information in the doc. Thank you for pointing it.

Comment (malvikasharan) [inline book/website/community-handbook/persona/persona-contributions.md:96]:
```suggestion
**Acknowledgement:** He is acknowledged in the Contributors table with a 👀 (`review`), 🤔 (`idea`) and 🖋 (`content`) emoji.
He is also named as an author on the last release of the book.
```

File: book/website/community-handbook/persona/persona-contributions.md [hunk 27]
```diff
@@ -0,0 +1,309 @@
...
+**Acknowledgement:** Ishan is acknowledged in the Contributors table with the 🤔 (`idea`) emoji.
+
+**4.** *Designing and writing a chapter*: After the same event as above, Mafalda completed the chapter that she brainstormed with Ishan.
+She submitted her contribution as a pull request to the Turing Way and responded to the review comments.
+
+**Acknowledgement:** She is acknowledged in the Contributors table with 🤔 (`idea`) and 🖋 (`content`) emoji.
+
+### Accessibility
+
+**Overview:** Contributors reporting or working on accessibility issues.
```
Comment (malvikasharan) [inline book/website/community-handbook/persona/persona-contributions.md:116]:
```suggestion
Contributors reporting or working on accessibility issues.
```

File: book/website/community-handbook/persona/persona-contributions.md (hunk 25 above)
Comment (malvikasharan) [inline book/website/community-handbook/persona/persona-contributions.md:277]:
```suggestion
We encourage creating video content or animations, record online interactive discussions or link any relevant videos our contributors may have created in the past that can be used as a learning tool and enhance the quality of _The Turing Way_ content.
```

**Comment by malvikasharan** [comment]
Thank you so much Rachael. This was a massive chapter - I am amazed by your high-efficiency review. 🙏🌸

**Comment by malvikasharan** [comment]
@rainsworth based on my discussion with Kirstie, I have moved the contributions file to the acknowledgment chapter, and contributor's file drafted by you to persona and pathway chapter in the project design. Since you have already reviewed both the chapters, I am tagging you for the final approval of this restructuring within the book.

**Comment by cassgvp** [comment]
Love the detail in these examples. The original Mozilla material I was working with suggested I take "3-4 minutes" sketching out a persona and the same amount of time for a pathway. It's taken me 2 days! Completing the exercise slowly and thoroughly has been so useful in helping me to spot gaps in my material and engagement strategy. This looks like an excellent resource to help people give this exercise the attention it deserves. The [personas and pathways I created](https://cassgvp.github.io/WIN-Open-Neuroimaging-Community/docs/community/personas-and-pathways.html) are here. I will definitely add an action to revisit them based on The Turing Way structure, probably as a "good first issue"!

**Comment by malvikasharan** [comment]
Thanks @cassgvp. I will have a look at your examples and maybe include them for non GitHub examples. 🌺

P.S. The first persona vaguely matches you 😱😱

**Review by rainsworth — APPROVED** [review]
Looks great, thanks @malvikasharan 🎉

**Comment by malvikasharan** [comment]
Thank you @rainsworth 🎉 It live now: https://the-turing-way.netlify.app/project-design/persona/persona-contributors.html. 💖

**Pull request merged**